src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -149,6 +149,9 @@ private ConfigProperties() {
     // Messaging
     public static final String CPM_PROVIDER = "candlepin.messaging.provider";
 
+    // OCSP stapling
+    public static final String SSL_VERIFY = "candlepin.sslverifystatus";
```

Comment:
Can you add it to `config/candlepin/candlepin.conf.template` somehow? E.g. It could be mentioned in comment.

Reply:
So added

Reply:
btw, adding this to the template means this property would only be added to candlepin.conf if the sslverifystatus_enabled variable is set to true, which is the kind of thing that the deploy script usually does, but I don't see the corresponding change to deploy script. So if left as is, this property won't be added to candlepin.conf

Reply:
in addition to that though: there is no way for anyone to actually test this feature except in Hosted Prod, so I don't see the point of setting it to true?

Reply:
If @jirihnidek is ok with not including this in the .conf file, I can merge the PRs as they are

Reply:
@nikosmoum I think it is OK and the PR can be merged.